tests/backup/backup_helper.go

```diff
@@ -1187,7 +1196,7 @@ func backupSuccessCheck(backupName string, orgID string, retryDuration int, retr
 	if err != nil {
 		return false, err
 	}
-	backupStatus := (resp.GetBackup().GetStatus().Status == api.BackupInfo_StatusInfo_Success)
+	backupStatus := resp.GetBackup().GetStatus().Status == api.BackupInfo_StatusInfo_Success
 	log.Infof("Backup [%s] created successfully", backupName)
```

Comment:
Remove this log entry because if the condition is false then this should not get printed.

Reply:
done